### DISCUSSION TITLE
Default Link Width?

### DISCUSSION THREAD

**Discussion by JeremyGEA** [Blazor-Diagrams/Blazor.Diagrams#235] [Q&A] [answered]
How can I set the default link line width for when the user draws a new line?
I would expect something like: `Diagram.Options.Links.DefaultWidth` but I'm not seeing it.

Thanks!

**Comment by zHaytam** [comment, marked as answer]
Please use `Options.Links.Factory` in order to create the link however you want it to be

**Reply by JeremyGEA** [reply]
Can you give an example as to how to do this with the Options.Links.Factory? I don't see a width property.

**Reply by snehabihani** [reply]
var options = new BlazorDiagramOptions
                 {
                 
                     Links =
                     {
                         Factory = (diagram, port) =>
                         {
                             var linkModel = new LinkModel(port);
                             linkModel .Width = 5.0;
                           
                             return linkModel;
                         }
                     },
                  
                 };

**Reply by JeremyGEA** [reply]
Thanks!
